src/node/interfaces.cpp

```diff
@@ -486,11 +486,6 @@ class ChainImpl : public Chain
         const CChain& active = Assert(m_node.chainman)->ActiveChain();
         return active.GetLocator();
     }
-    bool checkFinalTx(const CTransaction& tx) override
-    {
-        LOCK(cs_main);
-        return CheckFinalTx(chainman().ActiveChain().Tip(), tx);
```

Comment:
> The time is set in CheckFinalTx

Thanks. Hell, default arguments really are the worst. https://github.com/bitcoin/bitcoin/blob/bd482b3ffebc68130f8a18dabf08ed1aff7ea159/src/validation.cpp#L207-L209